.github/workflows/test_inf2.yml

```diff
@@ -47,15 +47,3 @@ jobs:
         run: |
           source aws_neuron_venv_pytorch/bin/activate
           HF_TOKEN=${{ secrets.HF_TOKEN_OPTIMUM_NEURON_CI }} pytest -m is_inferentia_test tests/generation
-      - name: Run exporters tests
```

Comment:
could we name the file `test_inf2_common` and job name with `name: Optimum neuron / Test INF2 Common`?

Reply:
I agree test_inf2.yml is a poor name, but these are not really common tests: all decoder tests are included. I was about to call it test_inf2_decoder, but there are some CLI tests also, so I kept the name ...

Reply:
Oki doki 😤, then we keep it this way for now.